### PR TITLE
[Feature] Checking SELECT action for mv in new privilege framework

### PR DESCRIPTION
**Pull request by nshangyiming**
## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Also support check specified aciton for any mv in a database.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 32 / 34 (94.12%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/PrivilegeCheckerV2.java|14|16|87.50%|[241, 244]|
|:large_blue_circle:|com/starrocks/common/ErrorCode.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/privilege/PrivilegeManager.java|17|17|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java [hunk 1]
```diff
@@ -77,6 +77,8 @@ public enum ErrorCode {
             "There is no such grant defined for user '%s' on host '%s'"),
     ERR_TABLEACCESS_DENIED_ERROR(1142, new byte[] {'4', '2', '0', '0', '0'},
             "%s command denied to user '%s'@'%s' for table '%s'"),
+    ERR_MV_ACCESS_DENIED_ERROR(1142, new byte[] {'4', '2', '0', '0', '0'},
```
Comment (Astralidea) [inline fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java:80]: this error code same as previous 1142 better use different one.

Comment (nshangyiming) [reply fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java:80]: you are right, will fix it

**Review by Astralidea — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by nshangyiming** [comment]
run starrocks_fe_unittest

**Comment by nshangyiming** [comment]
run starrocks_fe_unittest

**Pull request merged**